.github/workflows/examples-providers.yml

```
@@ -130,6 +130,12 @@ jobs:
         run: wash build
         working-directory: examples/${{ matrix.project.lang }}/providers/${{ matrix.project.name }}
 
+      # Build required project if necessary
+      - name: Build dependent component if present
+        shell: bash
+        run: |
+          [[ -d component ]] && wash dev -p component/wasmcloud.toml
```

Should this be `wash build` instead of `wash dev`?
```suggestion
          [[ -d component ]] && wash build -p component/wasmcloud.toml
```

---

ah sorry you're a bit early, change was still going in